Update media links

Kissaki opened the pull request:
Only one of the dead video links is available on the web archive.
Remove the others.

fdncred commented:
I really don't want to get rid of the reedline ones because they're the only reference we have to the originals and some of those are available on the wayback machine.

Kissaki commented:
We could put them into a separate section / add a note about that, and we could like the wayback machine then?

I can do that if you think that's viable/reasonable.

fdncred commented:
yup, i'm fine with a separate section

Kissaki commented:
You talked about the second block, the reedline videos. What about the first block with various dead video links?

fdncred commented:
I'm fine with keeping them all in another section, or even another file. I'm also fine with just keeping the reedline ones. It's up to you.

Kissaki commented:
I reduced this PR. None of these are available in the Internet Archive Wayback Machine.

Kissaki commented:
Actually, the way I checked was actually wrong (URL escaping :( ) so they may still be available

Kissaki committed:
Update media links

Only one of the dead video links is available on the web archive.
Remove the others.

fdncred commented:
Thanks
